contents/docs/ai-engineering/wizard.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Wizard 
+---
+
+import WizardInstall from '../getting-started/_snippets/wizard.mdx'
+
+The PostHog wizard helps you instantly add PostHog to your application codebase using AI. It's an agentic CLI tool that handles the entire PostHog installation and integration process on your behalf.
+
+```bash
+npx @posthog/wizard
```

Comment:
Is there an us/eu thing we show account for here? If it works fine without, all good.